store/storage/storage_bench_test.go

```diff
@@ -17,13 +19,13 @@ import (
 
 var (
 	backends = map[string]func(dataDir string) (store.VersionedDatabase, error){
-		"rocksdb": func(dataDir string) (store.VersionedDatabase, error) {
+		"rocksdb_versiondb_opts": func(dataDir string) (store.VersionedDatabase, error) {
 			return rocksdb.New(dataDir)
 		},
-		"pebbledb": func(dataDir string) (store.VersionedDatabase, error) {
+		"pebbledb_default_opts": func(dataDir string) (store.VersionedDatabase, error) {
 			return pebbledb.New(dataDir)
 		},
-		"btree(sqlite)": func(dataDir string) (store.VersionedDatabase, error) {
+		"btree_sqlite": func(dataDir string) (store.VersionedDatabase, error) {
```

Comment:
why do we keep the btree prefix to the sqlite, but not adding lsm prefix to the other dbs?

Reply:
My intention was to  indicate that it's not a direct BTree implementation, but rather an existing storage engine that utilizes BTrees under the hood.

I can add the type to all of them if you'd like.

Reply:
I mean we are just benchmarking sqlite, not btree, we can't simply benchmark btree without referring to a specific implementation, and there are so many design decisions a db implementation need to make.